Sources/Modules/FilesModule.swift

```diff
@@ -1188,5 +1188,54 @@ public class FilesModule {
         }
     }
 
+    /// Creates a zip of multiple files and folders.
+    ///
+    /// - Parameters:
+    ///   - name: The name of the zip file to be created
+    ///   - items: Array of files or folders to be part of the created zip
+    ///   - completion: Returns a standard ZipDownload object or an error
+    public func createZip(
+        name: String,
+        items: [[String: String]],
```

Comment:
May be worth trying to pass in Items with a custom object like shown [here](https://github.com/box/box-ios-sdk/blob/master/docs/usage/metadata.md#create-metadata-template) for metadata.